### PR TITLE
added support for detecting if you already own the course; added support for vscode including debugging

### PR DESCRIPTION
**Pull request by udemyhelper**
added support for detecting if you already own the course; added support for vscode including debugging

### CODE REVIEW

File: core/udemy.py [hunk 1]
```diff
@@ -97,7 +97,14 @@ def redeem(self, url: str) -> None:
         element_present = EC.presence_of_element_located(
             (By.XPATH, buy_course_button_xpath)
         )
-        WebDriverWait(self.driver, 10).until(element_present).click()
+
+        button = WebDriverWait(self.driver, 10).until(element_present)
+
+        if button.text == "Go to course":
```
Comment (cullzie) [inline core/udemy.py:103]: This won't work for all languages

**Review by cullzie — COMMENTED** [review]
There is already a check for this in develop branch:
https://github.com/aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE/blob/develop/core/udemy.py#L88-L94

**Comment by fakeid30** [comment]
@cullzie this is totally random but can you please send me an email to junayeed@protonmail.com , I have looked back and forth to find an email of yours but failed. I have something important to share with you. 

Thanks.

**Comment by fakeid30** [comment]
@udemyhelper any update on this ?

**Comment by fakeid30** [comment]
@cullzie i think we close this ?

**Comment by fakeid30** [comment]
@udemyhelper @nothingmn hello, this needs some improvement :)

**Pull request closed without merging**